src/components/CalibrationManagerCells.tsx

```diff
@@ -0,0 +1,78 @@
+import {
+    ModalBody, 
+    ModalContent,
+    ModalFooter, 
+    ModalHeader, 
+    ModalCloseButton,
+    Button,
+    Th,
+    Td,
+    Link,
+} from "@chakra-ui/react";
+import { MinUser } from "shared/User";
+import trpc from "trpc";
+import ModalWithBackdrop from "./ModalWithBackdrop";
+import UserSelector from "./UserSelector";
+import { useState } from "react";
+import { formatUserName } from "shared/strings";
+import { MdEdit } from 'react-icons/md';
+import { Calibration as SharedCalibration } from 'shared/Calibration';
+
+export function CalibrationManagerHeaderCells() {
+    return <>
+        <Th>面试主管</Th>
+    </>;
+}
+
+export function CalibrationManagerCells({ calibration, refetch }: {
+    calibration: SharedCalibration,
+    refetch: () => void,
+}) {
+    const [ editing, setEditing ] = useState<boolean>(false);
+    return <>
+      {editing && <ManagerEditor calibration={calibration} 
+        manager={calibration.manager}
+        onClose={() => setEditing(false)} refetch={refetch} />}
+
+      <Td><Link onClick={() => setEditing(true)}>
+        {calibration.manager ?
+           formatUserName(calibration.manager.name)
+        :
+        <MdEdit />
```

Comment:
indentation:
```
{... ?
  format...
  :
  <MdEdit...
}
```

or 

`{calibra... ? format... : <Md...}` all in one line if they don't exceed 80 chars